.travis.yml

```diff
@@ -83,7 +83,7 @@ matrix:
 
     # common distributions:
     # Debian Wheezy
-    - python: 2.7
+    - python: 3.8
```

Comment:
This configuration should also be removed if python 2 support is removed.

Reply:
OK configuration removed in next commit. Thx